src/inventory/inventory.service.ts

```diff
@@ -34,8 +36,14 @@ export class InventoryService {
     }
   }
 
-  async findAll() {
-    return this.productItemModel.find({});
+  async findAll(args: FindProductItemArgs): Promise<ProductItem[]> {
+    const { first, skip, orderBy } = args;
+    console.log('args', args);
```

Comment:
debug log?

Reply:
Kind of, I plan to add proper logging (persistent) later

Reply:
ok